README.md

```diff
@@ -23,6 +23,124 @@ Or install it yourself as:
 $ gem install activejob-web
 ```
 
+
+## Configuring Activejob Web approvers and executors
+
+By default, Activejob Web uses the `User` class as the model for job approvers and executors. If you need to customize these classes, you can do so in the initializer file located at `config/initializers/activejob_web.rb`.
+
+For example, if your model to configure is 'Author', you can set the job approvers and executors classes as follows:
+
+```ruby
+# config/initializers/activejob_web.rb
+
+Activejob::Web.job_approvers_class = 'Author'
+Activejob::Web.job_executors_class = 'Author'
+```
+where, `job_approvers_class` and `job_executors_class` were defined as `mattr_accessors` in Activejob Web gem.
+
+## Configuration for Authentication
+The ActivejobWeb Gem integrates with the authentication system used in your Rails application. To set up authentication, create a helper file named `authentication_helper.rb` with the `my_app_current_user` method:
+
+```ruby
+# app/helpers/user_helper.rb
+
+module AuthenticationHelper
+  def my_app_current_user
+    # Specify the current user here
+    User.find(current_user.id)
```

Comment:
@GowthamK01 Shall we use`current_user` itself here?

Reply:
@gomathi-mallow Have checked and changed `User.find(current_user.id)` to `current_user`.